integration-cli/docker_cli_pause_test.go

```diff
@@ -57,10 +57,3 @@ func (s *DockerSuite) TestPauseMultipleContainers(c *check.C) {
 		c.Assert(actions[len(actions)-1], checker.Equals, "unpause")
 	}
 }
-
-func (s *DockerSuite) TestPauseFailsOnWindows(c *check.C) {
```

Comment:
This test can stay (rename accordingly) with something like `testRequires(c, DaemonIsWindows, !IsPausable) to validate that Windows Server containers can't be paused.